tests/test_filter.py

```diff
@@ -0,0 +1,41 @@
+from functools import partial
```

Comment:
np: unused import

Reply:
Oh, sorry - I meant "np" as in "nitpick", not "numpy" :D 
The line that's unused is `from functools import partial`

Reply:
Ok solved